4.0/crud-operation-list-entries.md

```diff
@@ -116,7 +116,32 @@ Exporting the DataTable to PDF, CSV, XLS is as easy as typing ```$this->crud->en
 
 ![Backpack CRUD ListEntries Details Row](https://backpackforlaravel.com/uploads/docs-4-0/operations/listEntries_export_buttons.png)
 
->**Please note that when clicked, each button will export the _currently visible_ table.** You can use the "visibility" button, and the "Items per page" dropdown to manipulate what is inside the export.
+**Please note that when clicked, the button will export** 
```

Comment:
Excellent addition to the docs, thank you @pxpm ! This hasn't been properly documented until now, you're right.